frontera/strategy/__init__.py

```diff
@@ -147,3 +143,20 @@ def refresh_states(self, requests):
         :param requests: list(:class:`Request <frontera.core.models.Request>`)
         """
         self._states_context.refresh_and_keep(requests)
+
+    def request_error(self, request, error):
+        """
+        DEPRECATED.
```

Comment:
shouldn't it show a warning if it is deprecated?

Reply:
It has to be resolved some other way. I'll investigate it